src/js/abstract/hocs/with-update.js

```diff
@@ -144,6 +146,11 @@ const withUpdate = Base =>
       } else { // delete attribute
         this[key] = null;
       }
+
+      // if value is updated, we presume that an axa on change event have to be triggered
+      if (name === 'value' && newValue !== null && oldValue !== newValue) {
```

Comment:
Check for updating is done above by `shouldUpdateCallback`.

Hmh do we really want to fire this for all value attributes?